### DISCUSSION TITLE
Zipkin tracing

### DISCUSSION THREAD

**Discussion by guptadeepali39** [Kong/kong#8284] [Help] [answered]
Hi,

I am enabling zipkin plugin for all my routes. Now traces are working fine. But I want to know if we can get trace_id and span_id in response when we hit any api?

**Comment by kikito** [comment, marked as answer]
Hello, the Zipkin plugin sets the tracing headers on the requests for upstream, but not on the responses it gets back from the upstream service (unless the upstream service includes such headers). Perhaps this could be added as an option. Feel free to open a Pull Request!
